ui/app/templates/components/line-chart.hbs

```diff
@@ -0,0 +1,28 @@
+<svg>
+  <defs>
+    <linearGradient x1="0" x2="0" y1="0" y2="1" class="{{chartClass}}" id="{{fillId}}">
+      <stop class="start" offset="0%" />
+      <stop class="end" offset="100%" />
+    </linearGradient>
+    <clipPath id="{{maskId}}">
+      <path class="fill" d="{{area}}" />
+    </clipPath>
+  </defs>
+  <g class="y-gridlines gridlines" transform="translate({{yAxisOffset}}, 0)"></g>
+  <g class="canvas {{chartClass}}">
+    <path class="line" d="{{line}}" />
+    <rect class="area" x="0" y="0" width="{{yAxisOffset}}" height="{{xAxisOffset}}" fill="url(#{{fillId}})" clip-path="url(#{{maskId}})" />
+    <rect class="hover-target" x="0" y="0" width="{{yAxisOffset}}" height="{{xAxisOffset}}" onhover={{action updateActiveDatum}} />
```

Comment:
Why both the onhover here and the mouseenter/move in the d3 binding in the JS?

Reply:
Oh, oops. This onhover in the template should be deleted. Glimmer doesn't support binding events on svg elements; at least not in 2.18.